master/buildbot/reporters/bitbucket.py

```diff
@@ -47,13 +49,26 @@ def reconfigService(self, oauth_key, oauth_secret,
         if base_url.endswith('/'):
             base_url = base_url[:-1]
 
-        self._base_url = base_url
-        self._oauth_url = oauth_url
-        self._auth = (oauth_key, oauth_secret)
+        self.http = yield httpclientservice.HTTPClientService.getService(
+            self.master, base_url)
+
+        self.oauthhttp = yield httpclientservice.HTTPClientService.getService(
+            self.master, oauth_url, auth=(oauth_key, oauth_secret))
 
     @defer.inlineCallbacks
     def send(self, build):
         results = build['results']
+        oauth_request = yield self.oauthhttp.post("",
+                                                  json=_GET_TOKEN_DATA)
+        if oauth_request.code == 200:
+            content_json = yield oauth_request.json()
+            token = content_json['access_token']
+        else:
+            content = yield oauth_request.content()
+            log.error("{}: unable to authenticate to Bitbucket {}".format(
```

Comment:
this module uses the new logger api, which log.error is an indication of loglevel